proposals/0157-mobile-choice-manager.md

```diff
@@ -28,35 +28,74 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface SDLChoiceCell
 
-@property (copy, nonatomic, readonly) NSString *title;
-@property (copy, nonatomic, readonly, nullable) NSString *text;
-@property (copy, nonatomic, readonly, nullable) NSString *subText;
-@property (copy, nonatomic, readonly) NSArray<NSString *> *voiceCommands;
+@property (copy, nonatomic, readonly) NSString *text;
+@property (copy, nonatomic, readonly, nullable) NSString *secondaryText;
+@property (copy, nonatomic, readonly, nullable) NSString *tertiaryText;
+@property (copy, nonatomic, readonly, nullable) NSArray<NSString *> *voiceCommands;
 @property (strong, nonatomic, readonly, nullable) SDLArtwork *artwork;
 @property (strong, nonatomic, readonly, nullable) SDLArtwork *secondaryArtwork;
 
+- (instancetype)initWithText:(NSString *)text;
+- (instancetype)initWithText:(NSString *)text artwork:(nullable SDLArtwork *)artwork voiceCommands:(nullable NSArray<NSString *> *)voiceCommands;
+- (instancetype)initWithText:(NSString *)text secondaryText:(nullable NSString *)secondaryText tertiaryText:(nullable NSString *)tertiaryText voiceCommands:(nullable NSArray<NSString *> *)voiceCommands artwork:(nullable SDLArtwork *)artwork secondaryArtwork:(nullable SDLArtwork *)secondaryArtwork;
+
+@end
+
+NS_ASSUME_NONNULL_END
+```
+
+```objc
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SDLChoiceCellGroup
```

Comment:
This is new... trying to follow the idea behind it... Could you please add a description of what this class is about?